src/libs/actions/Report.js

```diff
@@ -519,9 +519,10 @@ function fetchIOUReportByID(iouReportID, chatReportID, shouldRedirectIfEmpty = f
  * fetching the iouReport and therefore should only be called if we are certain that the fetched iouReport is currently
  * open - else we would overwrite the existing open iouReportID with a closed iouReportID.
  *
- * Examples of usage include 'receieving a push notification', or 'paying an IOU', because both of these cases can only
- * occur for an iouReport that is currently open (notifications are not sent for closed iouReports, and you cannot pay a
- * closed IOU).
+ * Examples of correct usage include 'receieving a push notification', or 'paying an IOU', because both of these cases can only
+ * occur for an iouReport that is currently open (notifications are not sent for closed iouReports, and you cannot pay a closed
+ * IOU). Send Money is an incorrect use case, because these IOUReports are never associated with the chatReport and this would
+ * prevent outstanding IOUs from showing.
```

Comment:
Honestly this logic is super confusing and should be done on the backend without F.E needing to do this. I'm not going to do that though, as we're about to refactor our APIs for offline -- and because this will probably need to change anyway as part of Manual Requests. 

I'll do this at some point